NextcloudTalk/PollVotingView.swift

```diff
@@ -234,12 +234,12 @@ import UIKit
     override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         if section == PollSection.kPollSectionOptions.rawValue {
             let votes = poll?.numVoters ?? 0
-            let votesString = votes == 1 ? NSLocalizedString("vote", comment: "1 vote in a poll") : NSLocalizedString("votes", comment: "{number} votes in a poll")
-            let resultsString = NSLocalizedString("Results", comment: "")
+            let votesString = String(format: NSLocalizedString("%d votes", comment: "Votes in a poll"), votes)
```

Comment:
Shouldn’t `localizedStringWithFormat` be used here?

Reply:
Ivan will fix the stuff today. We mostly merged it to test if the syncing to transifex was working or needed more work somewhere.

Reply:
`localizedStringWithFormat` definition says:

```
Returns a string created by using a given format string as a template into which the remaining argument values are substituted according to the current locale.
```

So yes, let's change it in case the argument needs to be localized for some languages ;)